assets/js/base/hooks/use-typography-props.ts

```diff
@@ -2,30 +2,53 @@
 /**
  * External dependencies
  */
-import { isObject } from '@woocommerce/types';
+import { isFeaturePluginBuild } from '@woocommerce/block-settings';
+import { isObject, isString } from '@woocommerce/types';
 import { parseStyle } from '@woocommerce/base-utils';
 
+type WithClass = {
+	className: string;
+};
+
 type WithStyle = {
 	style: Record< string, unknown >;
 };
 
-export const useTypographyProps = ( attributes: unknown ): WithStyle => {
+export const useTypographyProps = (
+	attributes: unknown
+): WithStyle & WithClass => {
+	if ( ! isFeaturePluginBuild() ) {
```

Comment:
We need to support font size and line height in WC core, as they are not experimental.

Thinking about this, IMO `useTypographyProps()` shouldn't check whether it's WC core or the feature plugin. That should be checked at the block level when defining which styles are supported. What do you think?

Reply:
Right. Good point. I agree. I'll remove the check. I'll also check and see if all blocks utilizing `useTypographyProps()` include the necessary check.